app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb

```diff
@@ -0,0 +1,109 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach art and design</h2>
+     <div class="inset">
+    <p>Teaching art and design gives pupils the technical skills needed to express their ideas through a range of materials and forms of media. You'll help pupils develop their own skills by encouraging them to critically review their work. Your pupils will develop an appreciation of art and design, and of the diverse ideas and perspectives of others.</p>
+    <p>In addition to the personal rewards of teaching art and design, there are practical benefits too, including:</p>
+      <ul>
+        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teacher-pay">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
+        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme">generous and secure pension</a></li>
+        <li>more holiday days than people in many other professions</li>
+        <li>opportunities for progression</li>
+      </ul>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
+    name: "Chris Farr, teacher training adviser",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Teaching art and design opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from fashion and textiles, to illustration and video game design.</p>
+      <p>By thinking creatively, your pupils could make significant contributions to these diverse and impactful areas.</p>
+      <p>Your pupils will learn more from you than you might think.  As their teacher, you'll guide them though the knowledge and skills they need to show their understanding of art and design. You'll teach them valuable life skills such as critical thinking, adaptability and communication.</p>
+  </section>
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-art-and-design-programmes-of-study/national-curriculum-in-england-art-and-design-programmes-of-study">national curriculum for art and design</a>.</p>
```

Comment:
Should this open in the same tab? Or is that on a future ticket?

Reply:
@emmaachesongray future ticket, Jen can update all the links at once I think